### PR TITLE
Update README for services

### PR DESCRIPTION
**Pull request by rimrakhimov**
Update Readme for the root directory - extract common information into root. Update Readme for smart-contract-verifier

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,16 +21,59 @@ A set of services used by [Blockscout](https://blockscout.com/) blockchain explo
...
+
+Alternatively, you can build your own docker images or compile them directly from sources. 
+Some of such options are described in [build](docs/build.md).
+
+## Project Layouts
+
+Most of the projects consist of 3 main parts:
+1. `{service-name}-proto` - defines the gRPC proto file with all API related data.
+   Defines mapping HTTP/JSON requests and their parameters to those gRPC methods.
+2. `{service-name}-logic` - the crate with the implementation of the main business logic.
```
Comment (bragov4ik) [inline README.md:66]: Should we mention that sometimes it's called `{service-name}`? Like currently in `stats`, for example

File: README.md [hunk 2]
```diff
@@ -21,16 +21,59 @@ A set of services used by [Blockscout](https://blockscout.com/) blockchain explo
...
+See full list of ENVs and their description inside service directories.
+
+```shell
+docker run -p 8050:8050 --env-file <path-to-your-env-file> ghcr.io/blockscout/{service-name}:latest 
+```
+
+Alternatively, you can build your own docker images or compile them directly from sources. 
+Some of such options are described in [build](docs/build.md).
+
+## Project Layouts
```
Comment (bragov4ik) [inline README.md:61]: maybe also mention `justfile`s that are quite useful & present in most of the services

File: docs/build.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+# Build
+
+## Using docker
+Each service contains a Dockerfile in the service root directory.
+
+## Building from source
+
+### Preparation
+
+1. Install rustup from rustup.rs.
```
Comment (bragov4ik) [inline docs/build.md:10]:
```suggestion
1. Install rustup from [[rustup.rs](https://rustup.rs/)].
```

File: docs/common-envs.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# Common Environment Variables
```
Comment (bragov4ik) [inline docs/common-envs.md:1]: Shall we mention how to deduce env names from config structs?

**Review by bragov4ik — APPROVED** [review]

File: docs/common-envs.md (hunk 4 above)
Comment (rimrakhimov) [reply docs/common-envs.md:1]: Yeah, that probably a good idea. Do you think `common-envs` is a good place for that?

File: docs/common-envs.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+| `{SERVICE_NAME}__SERVER__HTTP__CORS__ALLOWED_METHODS`          |          | A list of methods which allowed origins can perform                              | `PUT, GET, POST, OPTIONS, DELETE, PATCH` |
+| `{SERVICE_NAME}__SERVER__HTTP__CORS__ALLOWED_CREDENTIALS`      |          | Allow users to make authenticated requests                                       | `true`                                   |
+| `{SERVICE_NAME}__SERVER__HTTP__CORS__MAX_AGE`                  |          | Sets a maximum time (in seconds) for which this CORS request may be cached       | `3600`                                   |
+| `{SERVICE_NAME}__SERVER__HTTP__CORS__BLOCK_ON_ORIGIN_MISMATCH` |          | Configures whether requests should be pre-emptively blocked on mismatched origin | `false`                                  |
+| `SMART_CONTRACT_VERIFIER__SERVER__GRPC__ENABLED`               |          | Enable GRPC API server                                                           | `false`                                  |
```
Comment (k1rill-fedoseev) [inline docs/common-envs.md:17]: Why it's not `{SERVICE_NAME}` everywhere?

Comment (rimrakhimov) [reply docs/common-envs.md]: By mistake. Fixed, thanks

File: smart-contract-verifier/README.md [hunk 6]
```diff
@@ -1,82 +1,45 @@
...
+No additional dependencies
+
+## How to enable
+Set the following ENVs on blockscout instance:
+- `MICROSERVICE_SC_VERIFIER_ENABLED=true`
+- `MICROSERVICE_SC_VERIFIER_URL={service_url}`
+- `MICROSERVICE_SC_VERIFIER_TYPE=sc_verifier`
+
+## Envs
+Here we describe only service specific variables. Variables common for all services can be found [here](../docs/common-envs.md).
```
Comment (k1rill-fedoseev) [inline smart-contract-verifier/README.md:19]: GPT proof-reading
```suggestion
Here, we describe variables specific to this service. Variables common to all services can be found [here](../docs/common-envs.md).
```

File: smart-contract-verifier/README.md [hunk 7]
```diff
@@ -1,82 +1,45 @@
...
-
-## Start
-For the details of how to run the service, go into corresponding
-transport protocol layer description:
-- [smart-contract-verifier-server](./smart-contract-verifier-server/README.md)
+**Smart-contract verifier** - service for verification of EVM based contracts. 
+Its basic idea is to accept bytecode to be verified and potential source files as input and return whether those files and bytecode correspond to each other.
+
+Is a backbone service for everything related to smart-contract verification in blockscout.
+Is required if you want to have smart-contract verification functionality on your instance.
```
Comment (k1rill-fedoseev) [inline smart-contract-verifier/README.md:7]: GPT proof-reading, wdyt?

```suggestion
**Smart-contract verifier** is a service for verifying EVM-based contracts. The primary function of this service is to receive bytecode and potential source files as inputs and determine whether the files and the bytecode correspond to each other.

This service serves as the core component for all activities related to smart-contract verification in BlockScout. It is essential for enabling smart-contract verification functionality on your instance.
```

Comment (rimrakhimov) [reply smart-contract-verifier/README.md:7]: Applied, thanks!

**Review by k1rill-fedoseev — APPROVED** [review]

**Review by sevenzing — APPROVED** [review]

**Pull request merged**